### ISSUE TITLE
Incorrect number of fields 

### ISSUE THREAD

**Issue by tsanchev** [DSB/MySQLDumper#16]
![image](https://cloud.githubusercontent.com/assets/8882098/22740529/e2d0e9fa-ee18-11e6-9f04-e00f80836807.png)

the actual number of recordrs is 776

**Comment by mateuszjarzewski** [comment]
Yes, this is cuz InnoDB is slow when counting total number of rows, so they just get the value from systemic MySQL table INNODB_SYS_INDEXES. 
But that counter is ofully inaccurate :(

Take a look into file: inc/functions_dump.php line 80:

```
$erg=mysql_fetch_array($res);
// Get nr of records -> need to do it this way because of incorrect returns when using InnoDBs
$sql_2="SELECT count(*) as `count_records` FROM `".$databases['Name'][$dump['dbindex']]."`.`".$erg['Name']."`";
```
By the way that is well known InnoDB limitation.
I'm afraid there is no way to fix this, maybe except running count(*) on all tables, but this will consume wast amount of resources and take forever to complete on big dataset.
